native/packages/react-native-backpack/libraries/Badge/index.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { requireNativeComponent } from 'react-native';
+
+type BadgeType =
+  | 'badgeTypeSuccess'
+  | 'badgeTypeWarning'
+  | 'badgeTypeDestructive'
+  | 'badgeTypeLight'
+  | 'badgeTypeInverse'
+  | 'badgeTypeOutline';
```

Comment:
Currently consumers can use `success`, `warning` etc, so this would be breaking, right? I'm thinking it may make more sense to keep the user-facing API the same and then convert `success => badgeTypeSuccess` internally.

Or alternatively, keep using `success` instead of `badgeTypeSuccess`. Is there a reason this changed? I'm assuming it's something to do with the Obj-C enum.

Reply:
Will fix